include/SDL3/SDL_joystick.h

```diff
@@ -309,6 +309,10 @@ extern DECLSPEC SDL_Joystick *SDLCALL SDL_GetJoystickFromPlayerIndex(int player_
 /**
  * Attach a new virtual joystick.
  *
+ * \param type Type of joystick
+ * \param naxes number of axes
+ * \param nbuttons number of buttons
+ * \param nhats number of hats
```

Comment:
Incosistent first letter capitalization, e.g. `Type` vs `number` .   Same in several other places too.

Reply:
done in https://github.com/libsdl-org/SDL/pull/7300
only this one.
indeed this is not consistent accross various SDL comment